go.mod

```diff
@@ -91,6 +91,7 @@ require (
 	github.com/joyent/triton-go v0.0.0-20190112182421-51ffac552869
 	github.com/keybase/go-crypto v0.0.0-20190403132359-d65b6b94177f
 	github.com/kr/pretty v0.1.0
+	github.com/kr/pty v1.1.3 // indirect
```

Comment:
Please go mod tidy. This is really annoying because for some reason right now this keeps getting added on build and removed on go mod tidy. (Are you on go 1.12.10? I only noticed it after upgrading to that, although it may be unrelated.) However, for the moment we're not committing build-time only changes since it'll be pulled right back out on the next tidy.

Reply:
My IDE (goland) also auto-adds it back, very annoying I agree.

Reply:
Try merging in master, it _should_ be fixed (I no longer get on my diffs).